### PR TITLE
Add PersistString Property

### PR DESCRIPTION
**Pull request by arbor95**
Simply setting this property with a recognizable string, par example if you have to handle more than one instance of a class.
.PersistString="something like the name of the instance"

LoadFromXML's DeserializeDockContent function can return the correct Docking.IDockContent by comparing the parameter with the PersistSting Property:

*.LoadFromXml(mStream, _DockContent, True) with _DockContent doing
Docking.DeserializeDockContent(AddressOf GetContentFromPersistString) and
function GetContentFromPersistString(ByVal persistString As String) as IDockContent can do like

If persistString.Equals(.PersistString) Then
Return 
with created in advance.

Sorry for the example in VB (what I use) instead of CS

Now hopefully without any messages

### CODE REVIEW

File: WinFormsUI/ThemeVS2003/VS2003Theme.cs [hunk 1]
```diff
@@ -55,23 +55,23 @@ internal static DockPanelSkin CreateVisualStudio2003()
             return skin;
         }
 
-        private class VS2003DockPaneStripFactory : DockPanelExtender.IDockPaneStripFactory
+        public class VS2003DockPaneStripFactory : DockPanelExtender.IDockPaneStripFactory
```
Comment (lextm) [inline WinFormsUI/ThemeVS2003/VS2003Theme.cs]: Why do you need to change the visibility of such classes? Please roll back.

File: WinFormsUI/ThemeVS2003/VS2003Theme.cs [hunk 2]
```diff
@@ -55,23 +55,23 @@ internal static DockPanelSkin CreateVisualStudio2003()
...
         {
             public DockPaneStripBase CreateDockPaneStrip(DockPane pane)
             {
                 return new VS2003DockPaneStrip(pane);
             }
         }
 
-        private class VS2003AutoHideStripFactory : DockPanelExtender.IAutoHideStripFactory
+        public class VS2003AutoHideStripFactory : DockPanelExtender.IAutoHideStripFactory
```
Comment (lextm) [inline WinFormsUI/ThemeVS2003/VS2003Theme.cs]: Please roll back this line.

File: WinFormsUI/ThemeVS2003/VS2003Theme.cs [hunk 3]
```diff
@@ -55,23 +55,23 @@ internal static DockPanelSkin CreateVisualStudio2003()
...
         {
             public AutoHideStripBase CreateAutoHideStrip(DockPanel panel)
             {
                 return new VS2003AutoHideStrip(panel);
             }
         }
 
-        private class VS2003DockPaneCaptionFactory : DockPanelExtender.IDockPaneCaptionFactory
+        public class VS2003DockPaneCaptionFactory : DockPanelExtender.IDockPaneCaptionFactory
```
Comment (lextm) [inline WinFormsUI/ThemeVS2003/VS2003Theme.cs]: Please roll back this line.

**Comment by arbor95** [comment]
> Is this included in the current release? I guess that it isn't, right?

you are right.
If you like you can use https://github.com/arbor95/dockpanelsuite .

**Comment by arbor95** [comment]
Hi lextm,
I don't know how to change the code of the pull request.

In my original fork I need the possibility for overwriting.
So for common usage you can change that 3 accessors from public to private.

thx arbor95

**Comment by lextm** [comment]
"I don't know how to change the code of the pull request."

Make a new commit with required changes, and then push it to the branch (arbor95:master in this case). Then GitHub will update the pull request accordingly.

**Comment by arbor95** [comment]
please merge pull request!

**Pull request closed without merging**